tests/src/kakarot/instructions/test_stop_and_math_operations.py

```diff
@@ -93,10 +80,10 @@ class TestMathOperations:
             ],
         )
         async def test__exec_math_operation(
-            self, math_operations, opcode, stack, expected_result
+            self, cairo_run, opcode, stack, expected_result
         ):
-            await math_operations.test__exec_math_operation(
-                opcode,
-                [int_to_uint256(v) for v in stack],
-                int_to_uint256(expected_result),
-            ).call()
+            stack = [tuple for value in stack for tuple in int_to_uint256(value)]
```

Comment:
don't use globals keyword as arg name, and especially here it's not a `tuple` anymore since you're iterating over it